### DISCUSSION TITLE
求助报错如下，急急急急求大佬相助，小弟不胜感激

### DISCUSSION THREAD

**Discussion by SashideMaria** [mikumifa/biliTickerBuy#197] [提问专区]
![{9({PO5L9PE 5WYT77HHOG6](https://github.com/mikumifa/biliTickerBuy/assets/139423891/4e5d99fc-1aed-4414-a4e4-28f2ccd69db6)
一打开就报错如下，点击如图连接没反应，复制粘贴到浏览器挂梯子也没反应

**Comment by mikumifa** [comment]
这个报错无所谓，对抢票没有影响，请看一下说明书

**Reply by SashideMaria** [reply]
好的，成功了，谢谢大佬！！
